tests/test_install_requires.py

```
@@ -38,47 +38,48 @@
-		self.requirements_file = """test_requirements.txt"""
+		self.requirements_file = "test_requirements.txt"
 
-	def tearDown(self):
+	def tearDown(self) -> None:
 		"""Clean up the temporary requirements file"""
 		try:
 			if os.path.exists(self.requirements_file):
 				os.remove(self.requirements_file)
 		finally:
-			super(TestParseRequirements, self).tearDown()
+			super(ParseRequirementsTestSuite, self).tearDown()
```

Ruff (0.8.2)

Use `super()` instead of `super(__class__, self)`

Remove `__super__` parameters

(UP008)